javav2/example_code/keyspaces/Readme.md

```diff
@@ -0,0 +1,79 @@
+- [Get data about a keyspace](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (getKeyspace command)
+- [Get data about a table](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (getTable command)
+- [List keyspaces](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (listKeyspacesPaginator command)
+- [List tables in a keyspace](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (listTablesPaginator command)
+- [Restore a table to a point in time](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (restoreTable command)
+- [Update a table](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (updateTable command)
+
+
+### Scenario 
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Get started with keyspaces and tables](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/keyspaces/src/main/java/com/example/keyspace/ScenarioKeyspaces.java) (multiple commands)
+
+## Run the Amazon Keyspaces Java files
+
+### Prerequisites
+
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html). 
+
+ These examples requires a **cassandra_truststore.jks** file to make a connection to Amazon Keyspaces.
+ For more information, see [Using a Cassandra Java client driver to access Amazon Keyspaces programmatically](https://docs.aws.amazon.com/keyspaces/latest/devguide/using_java_driver.html). 
+
+**Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
+
+ ## Test the Amazon Keyspaces Java files
+ 
+ ⚠️ Running the tests might result in charges to your AWS account.
+
+You can test the Java code example for Amazon Keyspaces by running a test file named **KeyspaceTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+
+Define these values in the test to successfully run the JUnit tests:
+
+- **fileName** - The name of the JSON file that contains movie data (you can get this file in the Github repo at resources/sample_file).
```

Comment:
- **fileName** - The name of the JSON file that contains movie data. (Get this file from the GitHub repo at resources/sample_file.)